corehq/apps/reports/dispatcher.py

```diff
@@ -184,7 +184,7 @@ def navigation_sections(cls, context):
                     else:
                         report_contexts.append({
                             'is_active': report.slug == current_slug,
-                            'url': report.get_url(domain=domain),
+                            'url': report.get_url(domain=domain, request=request),
```

Comment:
it looks like there are at least some references to `get_url` that aren't prepared to take an extra argument. for example here: https://github.com/dimagi/commcare-hq/blob/master/corehq/__init__.py#L155

@dimagi/team-commcare-hq does anyone have any opinions on this? (some custom reports want to override `get_url` with user context-specific information). i'm not a huge fan of adding more stuff to the api, but not sure i have a better proposal